helmfile/charts/networkpolicy-service/templates/harbor/allow-harbor-core.yaml

```diff
@@ -162,6 +162,7 @@ spec:
             component: jobservice
       ports:
         - port: 8080
+    {{- if and .Values.harbor.registries.ips .Values.harbor.registries.ports }}
```

Comment:
@viktor-f 
This is my solution for this bug
https://github.com/elastisys/compliantkubernetes-apps/issues/1363

Reply:
Looks good